plugins/Actions/VisitorDetails.php

```diff
@@ -252,6 +261,29 @@ public function extendActionDetails(&$action, $nextAction, $visitorDetails)
         unset($action['idlink_va']);
     }
 
+    private function shouldUseHttpsHost($idSite, $host)
+    {
+        static $siteUrlCache = [];
```

Comment:
is there any chance to not use static here as we can't reset it in tests? We could use the transient cache or so instead? Or might be even fine to just use a regular property as there's likely only one instance per request anyway maybe?

Reply:
transient cache should work as well...